### PR TITLE
bump docker to newer version

### PR DESCRIPTION
**Pull request by willysr**

### CODE REVIEW

**Comment by willysr** [comment]
@sbo-bot: build x86_64 system/runc

**Comment by willysr** [comment]
@sbo-bot: build x86_64 system/containerd

**Comment by willysr** [comment]
@sbo-bot: build x86_64 system/docker

**Comment by willysr** [comment]
@sbo-bot: build x86_64 system/docker-cli

**Comment by willysr** [comment]
@sbo-bot: build x86_64 system/buildah

**Comment by willysr** [comment]
@sbo-bot: build x86_64 system/podman

**Comment by willysr** [comment]
@sbo-bot: build x86_64 development/libnvidia-container

**Comment by willysr** [comment]
@sbo-bot: build x86_64 python/subuser

File: system/containerd/containerd.SlackBuild [hunk 1]
```diff
@@ -26,7 +26,7 @@
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=containerd
-VERSION=${VERSION:-1.7.14}
+VERSION=${VERSION:-1.7.20}
 GITHASH=${GITHASH:-dcf2847247e18caba8dce86522029642f60fe96b}
```
Comment (aclemons) [inline system/containerd/containerd.SlackBuild:30]: Can you bump the git hash (8fc6bcff51318944179630522a095cc9dbf9f353) too?

File: system/docker-cli/docker-cli.SlackBuild [hunk 2]
```diff
@@ -27,7 +27,7 @@
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=docker-cli
-VERSION=${VERSION:-24.0.9}
+VERSION=${VERSION:-27.1.1}
 GITHASH=${GITHASH:-293681613032e6d1a39cc88115847d3984195c24}
```
Comment (aclemons) [inline system/docker-cli/docker-cli.SlackBuild:31]: here too (63125853e3a21c84f3f59eac6a0943e2a4008cf6)

File: system/runc/runc.SlackBuild [hunk 3]
```diff
@@ -28,7 +28,7 @@
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=runc
-VERSION=${VERSION:-1.1.12}
+VERSION=${VERSION:-1.1.13}
 GITHASH=${GITHASH:-51d5e94601ceffbbd85688df1c928ecccbfa4685}
```
Comment (aclemons) [inline system/runc/runc.SlackBuild:32]: hash (58aa9203c123022138b22cf96540c284876a7910)

File: system/docker/docker.SlackBuild [hunk 4]
```diff
@@ -26,7 +26,7 @@
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=docker
-VERSION=${VERSION:-24.0.9}
+VERSION=${VERSION:-27.1.1}
 GITHASH=${GITHASH:-fca702d}
```
Comment (aclemons) [inline system/docker/docker.SlackBuild:30]: cc13f95

File: system/runc/runc.SlackBuild (hunk 3 above)
Comment (willysr) [reply system/runc/runc.SlackBuild:32]: done

File: system/docker-cli/docker-cli.SlackBuild (hunk 2 above)
Comment (willysr) [reply system/docker-cli/docker-cli.SlackBuild:31]: done

File: system/containerd/containerd.SlackBuild (hunk 1 above)
Comment (willysr) [reply system/containerd/containerd.SlackBuild]: done

File: system/docker/docker.SlackBuild (hunk 4 above)
Comment (willysr) [reply system/docker/docker.SlackBuild]: done

**Comment by willysr** [comment]
all direct dependencies are working fine with these updates, I am just waiting for the CI confirmation
if it goes well, you can merge it on your branch @aclemons

**Review by aclemons — APPROVED** [review]
LGTM - staged to my branch

**Pull request closed without merging**